Updating the ACL LOG doc in acl-log.md. (#2315)
redis/redis-doc: commands/acl-log.md

    9) "username"
   10) "someuser"
   11) "age-seconds"
   12) "8.038"
   13) "client-info"
   14) "id=3 addr=127.0.0.1:57275 laddr=127.0.0.1:6379 fd=8 name= age=16 idle=0 flags=N db=0 sub=0 psub=0 ssub=0 multi=-1 qbuf=48 qbuf-free=16842 argv-mem=25 multi-mem=0 rbs=1024 rbp=0 obl=0 oll=0 omem=0 tot-mem=18737 events=r cmd=auth user=default redir=-1 resp=2"
   15) "entry-id"
   16) (integer) 0
   17) "timestamp-created"
   18) (integer) 1675361492408
   19) "timestamp-last-updated"
   20) (integer) 1675361492408
```

> Each LOG is composed of the following fields of the corresponding acl log and their values:

Reviewer: ```suggestion
Each log entry is composed of the following fields:
```

Author: will make the necessary changes

> 1. `count`: The number of security events detected within a 60 second periods that are represented by this entry.

Reviewer: ```suggestion
1. `count`: The number of security events detected within a 60 second period that are represented by this entry.
```

> 2. `reason`: The reason for which the ACL security events were logged. Either `command`, `key`, `channel`, or `auth`.

Reviewer: ```suggestion
2. `reason`: The reason that the security events were logged. Either `command`, `key`, `channel`, or `auth`.
```
